### ISSUE TITLE
test one-category configurations without the delta-function ITD to see if they run

### ISSUE THREAD

**Issue by eclare108213** [CICE-Consortium/CICE#519]
Deprecating the delta-function ice thickness distribution is on hold, since it may be needed for one-category runs.  These simple tests could be run in CICE or possibly Icepack.  If they do run, we could go ahead and deprecate the delta-function ITD, otherwise we'll keep it.

**Comment by JFLemieux73** [comment]
@eclare108213,

I am going to use box2001 for this. By default we have:

kridge          = -1
ktransport      = -1

Is it what we want for these tests? I guess ridging does not make sense with only one category...

**Comment by eclare108213** [comment]
It's probably sufficient to test only with thermodynamics, provided you get enough ice growth/melt to change the category that the ice is in.  Please also try it with the ridging and transport turned on.  Even with one category, the ridging code still acts to maintain aice <= 1.

**Comment by JFLemieux73** [comment]
How do you suggest I test thermodynamics?  Box2001 would not work.

**Comment by eclare108213** [comment]
Good point.  In that case then just turn on ridging and transport...  If this doesn't work out, I'll set up a gx3 test to do it.  Thanks

**Comment by JFLemieux73** [comment]
I set ncat=1. I had to comment abort_list = trim(abort_list)//":4" in ice_init.F90 for it to work. 

It is not clear to me what should be the value of kcatbound when ncat=1. Anyway here are the exps I tried:

exp1:
kridge=-1, ktransport=-1, kcatbound=0 ... PASS

exp2:
kridge=1, ktransport=1, kcatbound=0 ... PASS

exp3:
kridge=1, ktransport=1, kcatbound=-1 ... PASS

All the tests pass.

**Comment by eclare108213** [comment]
Excellent.  Please, would you attach the diagnostic output files here, for each of those tests?  Thanks!

**Comment by JFLemieux73** [comment]
You mean the cice.runlog files?

**Comment by eclare108213** [comment]
yep

**Comment by JFLemieux73** [comment]
EXP 1 (kridge=-1, ktransport=-1, kcatbound=0)
[cice.runlog.201113-185039_EXP1.txt](https://github.com/CICE-Consortium/CICE/files/5538853/cice.runlog.201113-185039_EXP1.txt)

EXP2 (kridge=1, ktransport=1, kcatbound=0)
[cice.runlog.201113-185323_EXP2.txt](https://github.com/CICE-Consortium/CICE/files/5538858/cice.runlog.201113-185323_EXP2.txt)

EXP3 (kridge=1, ktransport=1, kcatbound=-1)
[cice.runlog.201113-185653_EXP3.txt](https://github.com/CICE-Consortium/CICE/files/5538860/cice.runlog.201113-185653_EXP3.txt)

**Comment by eclare108213** [comment]
The code runs in all of these cases, but I'm not sure what it's actually doing.  Until that's understood, I don't think we should deprecate the delta-function ITD (kitd=0).  For completeness in this discussion, I'll note that the following appears in all of the log files:

>  (input_data) ERROR: kitd incompatability: ncat=1 and kitd=1
>  (input_data) ERROR:   Remapping the ITD is not allowed for ncat=1.
>  (input_data) ERROR:   Use kitd = 0 (delta function ITD) with kcatbound = 0
>  (input_data) ERROR:   or for column configurations use kcatbound = -1


On a different subject:  Why is `arwt heat error` so big in these runs?
